src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -1,25 +1,125 @@
 import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useState, useEffect } from 'react';
 import Header from '../../components/Header';
 import styles from './LostAndFound.module.css';
+import { useTheme } from '@mui/material/styles'; // Import theme
+
+// Define the type for Missing Item
+type MissingItem = {
+  id: string;
+  dateLost: string;
+  location: string;
+  category: string;
+  description: string;
+  status: string;
+};
 
 const LostAndFound = () => {
+  const [activeReports, setActiveReports] = useState<MissingItem[]>([]);
+  const [pastReports, setPastReports] = useState<MissingItem[]>([]);
+  const [loading, setLoading] = useState(true);
+  const theme = useTheme(); // Access theme
+
+  // Mock Data for demonstration
+  const mockActiveReports: MissingItem[] = [
+    {
+      id: '1',
+      dateLost: '9/9/24',
+      location: 'Probably Bennett',
+      category: 'Electronics',
+      description: 'iPhone 15 pro max, with black case',
+      status: 'lost',
+    },
+    {
+      id: '2',
+      dateLost: '5/4/24',
+      location: 'Barrington',
+      category: 'Books',
+      description: 'Klara and the Sun',
+      status: 'lost',
+    },
+  ];
+
+  const mockPastReports: MissingItem[] = [
+    {
+      id: '3',
+      dateLost: '10/5/24',
+      location: 'KOSC',
+      category: 'Electronics',
+      description: 'MacBook Pro, gray',
+      status: 'found',
+    },
+    {
+      id: '4',
+      dateLost: '4/5/24',
+      location: 'Jenks',
+      category: 'Bottles/Mugs',
+      description: 'Blue hydroflask, with a sticker',
+      status: 'deleted',
+    },
+    {
+      id: '5',
+      dateLost: '11/13/23',
+      location: 'AJ Chapel',
+      category: 'Clothing/Shoes',
+      description: 'Purple vikings hoodie',
+      status: 'expired',
+    },
+  ];
+
+  useEffect(() => {
+    setLoading(true);
+
+    // Simulate loading the mock data
+    setTimeout(() => {
+      setActiveReports(mockActiveReports);
+      setPastReports(mockPastReports);
+      setLoading(false);
+    }, 1000); // Simulate a loading delay of 1 second
+  }, []);
+
+  const handleEdit = (reportId: string) => {
+    console.log(`Editing report: ${reportId}`);
+  };
+
+  const handleDelete = (reportId: string) => {
+    console.log(`Deleting report: ${reportId}`);
+  };
+
   return (
     <>
-      <Header></Header>
-      <Grid container justifyContent="center">
+      <Header />
+
+      <Grid container justifyContent="center" marginTop={3}>
         <Grid item sm={10}>
           <Card className={styles.card}>
-            <CardHeader className={styles.title} title="Gordon Lost and Found"></CardHeader>
+            <CardHeader
+              className={styles.title}
+              title={
+                <Typography variant="h4" align="center">
```

Comment:
This styling should be done in a separate CSS file using a class name in general, it's better to have the styling in one place rather than inline, makes it easier to find and change later